release-process.rst

```diff
@@ -66,13 +89,19 @@ Patch Version Number
    -  Bug fixes and security patches only
    -  Extensions support can't be removed (like move them to PECL)
    -  Backward compatibility must be kept (internals and userland)
-   -  ABI and API compatibility must be kept (internals)
+   -  ABI and internal API compatibility must be preserved, except in the case
+      of HIGH severity security issues where no other option is possible.
```

Comment:
RFC 2119 would use `MUST` here, but the except makes it a `SHOULD`. Maybe phrase it as:
```
- ABI and internal API compatibility SHOULD BE preserved for "high" severity security issues, and MUST BE preserved for all other security issues.
```

Reply:
this sounds better - fixed